src/grt/src/fastroute/src/DataType.h

```diff
@@ -119,14 +119,16 @@ typedef struct
   short status;
   short conCNT;
   short botL, topL;
-  short heights[6];
+  // heights and eID arrays size were increased after using PD
+  // to create the tree topologies. 
+  short heights[10];
```

Comment:
what is the significance of "10"?
this should have a constexpr int constant with a name describing it's function instead of some random contstant.